src/routes/Projects/routes/Project/components/Editor/Editor.js

```diff
@@ -141,7 +141,10 @@ export default class Editor extends Component {
 
   render () {
     return (
-      <div className={classes['container']} id={this.props.name} />
+      <section className={classes['editor']}>
+        <style>{'.CodeMirror,.CodeMirror-scroll,.firepad {overflow: hidden; overflow-y: scroll; position: relative; height: 100%; width: 100%;}'}</style>
```

Comment:
That's actually how I did it first, and it did not work. I think it's because Firepad attaches its stylesheet to the end of the document -- which overrides the overrides. We can use `!important` perhaps?